allennlp/common/util.py

```diff
@@ -361,20 +361,37 @@ def peak_memory_mb() -> float:
     Only works on OSX and Linux, returns 0.0 otherwise.
```

Comment:
Might be nice to update this to better describe the return value.  It's not obvious why the return value is a dictionary.